src/jobsautoreport/main.py

```diff
@@ -20,8 +20,17 @@ def main() -> None:
 
     client = OpenSearch(os_host, http_auth=(os_usr, os_pwd))
 
-    now = datetime.now()
-    a_week_ago = now - timedelta(weeks=1)
+    now = datetime.now(tz=timezone.utc)
+    test_end_time = datetime(
+        year=now.year,
+        month=now.month,
+        day=now.day,
+        hour=0,
+        minute=0,
+        second=0,
+        tzinfo=timezone.utc,
+    )
+    test_start_time = test_end_time - timedelta(weeks=1)
```

Comment:
```suggestion
    report_end_time = datetime.now(tz=timezone.utc).replace(hour=0, minute=0, second=0, microsecond=0)
    report_start_time = now - timedelta(weeks=1)
```
no need to have a temporary variable in this case